projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -415,19 +426,28 @@ export class UpdateChanges {
     }
 
     protected updateClassMembers(entryPath: string, memberChanges: MemberChanges) {
-        const langServ = this.getDefaultLanguageService(entryPath);
         let content = this.host.read(entryPath).toString();
         const changes = new Set<{ change; position }>();
         for (const change of memberChanges.changes) {
 
```

Comment:
```suggestion
```

Reply:
I assume this was about calculating `absPath` multiple times (while it would never change) and assigning the `langServ` multiple times per file?

Reply:
Actually it was just about the empty row😄